packages/dd-trace/src/appsec/rasp/fs-plugin.js

```diff
@@ -0,0 +1,99 @@
+'use strict'
+
+const Plugin = require('../../plugins/plugin')
+const { storage } = require('../../../../datadog-core')
+const log = require('../../log')
+
+const RASP_MODULE = 'rasp'
+const IAST_MODULE = 'iast'
+
+const enabledFor = {
+  [RASP_MODULE]: false,
+  [IAST_MODULE]: false
+}
+
+let fsPlugin
+
+function enterWith (fsProps, store = storage.getStore()) {
+  if (store && !store.fs?.opExcluded) {
```

Comment:
`!store.fs?.opExcluded` could be true in several situations:

- if `opExcluded` exists and its value is `false` (I think it is the intended check)
- if `opExcluded` does not exist in `fs` object
- if `fs` does not exist in `store` object

Are the two latter conditions ok to proceed with the if body? Or is it assure that they never will happen?

Reply:
yes, they are.
`enterWith` is NOT excluded if:
- `fs` doesn't exist -> probably it is a root fs operation so `enterWith` will create a new store with the `fs` attribute
- `fs.opExcluded` doesn't exist but `fs` does -> store has been created before by a root fs operation